packages/webapp/src/_app/hooks/box-queries.ts

```diff
@@ -1,7 +1,57 @@
-import { Query, useQuery } from "@edenos/common/dist/subchain";
+import { Query, usePagedQuery, useQuery } from "@edenos/common/dist/subchain";
 import dayjs from "dayjs";
-import { MemberAccountData } from "members";
+
 import { assetFromString } from "_app";
+import { MemberAccountData } from "members";
+
+export const usePagedMembers = (
+    pageSize: number = 20
+): ReturnType<typeof usePagedQuery> => {
+    const query = `{
+    members(@page@) {
+        pageInfo {
+          hasPreviousPage
+          hasNextPage
+          startCursor
+          endCursor
+        }
+        edges {
+            node {
+                account
+                createdAt
+                profile {
+                    name
+                    img
+                    social
+                }
+            }
+        }
+    }
+}
+    `;
+
+    const pagedResult = usePagedQuery<any>(
+        query,
+        pageSize,
+        (result) => result.data?.members.pageInfo
+    );
+
+    let formattedMembers: MemberAccountData[] = [];
+
+    if (pagedResult.result.data) {
+        const memberNodes = pagedResult.result.data.members.edges;
+        if (memberNodes) {
+            formattedMembers = memberNodes.map((member: any) =>
+                formatQueriedMemberAccountData(member.node)
+            );
+        }
+    }
+
+    return {
+        ...pagedResult,
+        result: { ...pagedResult.result, data: formattedMembers },
+    };
+};
```

Comment:
suggestion (non-blocking/next-prs): I think since we are starting to add more queries we should think about that idea of splitting the queries into their own domain folder. same for the regular queries.ts

Reply:
💯 I have that as a future action item in my description, actually.